client/landing/gutenboarding/hooks/use-on-site-creation.ts

```diff
@@ -18,15 +21,18 @@ import { recordOnboardingComplete } from '../lib/analytics';
 import { useSelectedPlan, useShouldRedirectToEditorAfterCheckout } from './use-selected-plan';
 import { clearLastNonEditorRoute } from '../lib/clear-last-non-editor-route';
 import { useOnboardingFlow } from '../path';
+import doesValueExist from 'calypso/my-sites/checkout/composite-checkout/lib/does-value-exist';
 
-const wpcom = wp.undocumented();
+const wpcomGetCart = ( cartKey: string ) => wp.req.get( `/me/shopping-cart/${ cartKey }` );
+const wpcomSetCart = ( cartKey: string, cartData: RequestCart ) =>
+	wp.req.post( `/me/shopping-cart/${ cartKey }`, cartData );
```

Comment:
I noticed the same code being used in `client/lib/signup/cart.ts#l19-21`. To keep things DRY, it's better to extract this into a _lib_.